core/src/com/po/fuck/model/constants/ConstantsLoader.java

```diff
@@ -0,0 +1,32 @@
+package com.po.fuck.model.constants;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConstantsLoader {
+    protected static Properties loadProperties(String filePath) {
+        try {
+            Properties properties = new Properties();
+            properties.load(new FileInputStream(filePath));
+            return properties;
+        } catch (IOException io) {
+            throw new RuntimeException("Unable to load properties file: " + filePath + " - " + io.getMessage());
+        }
+    }
+
+    protected static String loadConstant(String name, Properties properties) {
+        String value = properties.getProperty(name);
+        if(value == null)
+            throw new RuntimeException(name + " is null");
```

Comment:
Are you sure we don't want to have some constants to be null ?

Reply:
Why would we?

Reply:
Maybe null means to disable some feature. For example: `CHAT_COOLDOWN = null`

Reply:
Null is not intended